h2o-bindings/bin/gen_python.py

```diff
@@ -550,6 +550,28 @@ def _determine_vec_size(self):
 
             self.vec_size = self.pre_trained.dim[1] - 1;
         """
+    elif algo == "pca":
+        return """
+        def init_for_pipeline(self):
+            \"\"\"
+            Returns H2OPCA object which implements fit and transform method to be used in sklearn.Pipeline properly.
+
+            :returns: H2OPCA object
+            \"\"\"
+            from h2o.transforms.decomposition import H2OPCA
+            return H2OPCA(model_id=self.model_id,
```

Comment:
Are all the possible parameters propagated to H2OPCA?

We should make sure none of the parameters get lost.

Reply:
That's the question. Because if we want to keep both algos (`H2OPCA` and `H2OPrincipalComponentAnalysisEstimator`), a developer would have always checked if the parameters fit in both of implementation. But I understand and try to find a better solution. Thank you.